build.gradle

```diff
@@ -22,8 +22,8 @@ import de.itemis.mps.gradle.*
 import java.time.LocalDateTime
 
 
-task wrapper(type: Wrapper) {
-    gradleVersion '4.4.1'
+wrapper {
+    gradleVersion '5.1'
```

Comment:
Why 5.1? are there specific reasons not to update to 5.2?

Reply:
Na, there aren't. Just haven't seen that 5.2 is already available.

Reply:
Just FYI: I'll amend the next commit.